docs/Privacy/Privacy-Quickstart.md

```diff
@@ -0,0 +1,141 @@
+description: Pantheon private network with privacy enabled quickstart tutorial
+<!--- END of page meta data -->
+
+# Private Network with Privacy Enabled Quickstart Tutorial
+
+The Private Network with Privacy Enabled Quickstart runs a private network of Pantheon and Orion nodes managed by Docker Compose.
+It is an expanded version of the [Private Network Quickstart](../Tutorials/Private-Network-Quickstart.md). 
+
+You can use the [Block Explorer](../Tutorials/Private-Network-Quickstart.md#block-explorer), 
+make [JSON-RPC requests](../Tutorials/Private-Network-Quickstart.md#run-json-rpc-requests), and 
+create [transactions using Metamask](../Tutorials/Private-Network-Quickstart.md#creating-a-transaction-using-metamask)
+as described in the [Private Network Quickstart tutorial](../Tutorials/Private-Network-Quickstart.md). 
+This tutorial describes how to use the examples provided in the EEAJS library to [create and send private transactions](#send-private-transactions-and-read-values). 
+
+!!! important 
```

Comment:
Perhaps we could add this to the other quickstart too (another PR)

Reply:
Already done :-)